src/apis/exportBookmarks.js

```diff
@@ -1,3 +1,22 @@
+/**
+ * Returns a dictionary with page indices as keys and the bookmark text as the values
+ * @method WebViewer#exportBookmarks
+ * @example
+WebViewer(...)
+  .then(function(instance) {
+    // Save the annotation data for doc123
+    const bookmarks = instance.exportBookmarks();
+    const bookmarksString = JSON.stringify(bookmarks);
+    fetch('/server/bookmarksHandler.js?documentId=doc123', {
+      method: 'POST',
+      body: bookmarksString // written into a json file on server
+    });
+  });
+ */
+/**
```

Comment:
I think this should just be `*`, otherwise the `returns` won't be included in the documentation.

![Screen Shot 2019-11-22 at 12 30 04 PM](https://user-images.githubusercontent.com/25110092/69458357-f85bbe80-0d23-11ea-9cd1-8a21c5ef2eeb.png)

Reply:
Fixed now.